package.nls.json

```diff
@@ -384,7 +384,7 @@
     "DataScience.runAll": "Run all cells",
     "DataScience.runAbove": "Run cells above",
     "DataScience.runBelow": "Run cell and below",
-    "DataScience.toggleCellOutput": "Toggle output of all selected cells",
```

Comment:
Feels wrong to say All, I would have thought that's implied when you say `selected`.
Not something I've seen on other places.
@claudiaregio /cc

Reply:
I had pulled this from Jupyter but it appears they have updated it too. 

Let's change to "Notebook: Toggle Cell Output"

Reply:
Done:
![image](https://user-images.githubusercontent.com/30305945/116645073-db201c00-a929-11eb-96fe-689e4f0de36c.png)